src/sass/_instance_detail_snapshots.scss

```diff
@@ -26,8 +26,7 @@
     }
   }
 
-  th:last-child,
-  td:last-child {
+  .p-table__expanding-panel {
```

Comment:
I cannot find this referenced anywhere.

Reply:
When selecting a snapshot this hides the "x snapshots selected" line that causes the table to jump. The old css was causing a bug with the snapshot configuration, where it was hiding an important column. 
So I changed the CSS selector to fix it.
Not sure, it is ideal now, because on instances we have the multiselect just changed to have the pagination on top. I'll create a ticket to address the multiselect of snapshots as well.

Reply:
This `p-table__expanding-panel` CSS class does not seem to be used anywhere.

Reply:
It is coming in from the `MainTable` component in react components.

Reply:
Ok, that's the answer that I needed. :+1: